en/core-api/direct-debit.md

```diff
@@ -260,12 +385,40 @@ The response is in JSON format:
 }
 ```
 
-### 4. Handling Post-Transaction
+### 4. Handling post-transaction
+
+When the transaction status changes, you are directly notified about the changes in the transaction through redirect URL and also on merchant's backend. Midtrans sends HTTP notification to merchant's backend. This ensures that you are updated of the transaction status securely.
+
+HTTP POST request with JSON body will be sent to your *Payment Notification URL* configured on dashboard. 
+
+#### **Configuring Payment Notification URL**
 
-HTTP notification from Midtrans to Merchant backend will also be triggered on event of `transaction_status` getting updated, to ensure merchant is securely informed. Including if transaction success or expired (not paid). So apart of JSON result above, Merchant backend will be notified by Midtrans.
+To configure the Payment Notification URL, follow the steps given below.
+
+1. Login to your MAP account.
+
+2. On the Home page, go to **SETTINGS > CONFIGURATION**.
+
+   ​	*Configuration* page is displayed.
+
+3. Enter **Payment Notification URL**.
+
+4. Click **Update**.
+
+   A confirmation message is displayed.
```

Comment:
Check preceding spaces in transition lines after step 2 and 4. They do not seem consistent.